src/site/markdown/server/manage-webadmin.md

```diff
@@ -1754,6 +1755,108 @@ Response codes:
  - 404: The mail queue does not exist
  - 500: Internal error
 
+## Administrating DLP Configuration
+Webadmin allows Data Lost Prevention (DLP) configuration
+
+- [List DLP configuration by sender domain](List_dlp_configuration_by_sender_domain)
+- [Store DLP configuration by sender domain](Store_dlp_configuration_by_sender_domain)
+- [Remove DLP configuration by sender domain](Remove_dlp_configuration_by_sender_domain)
+
+### List DLP configuration by sender domain
+
+Retrieve a DLP configuration for corresponding `senderDomain`, a configuration contains list of configuration item
```

Comment:
Could you explain what is  sender domain?
Could you give me a possible example for a value?

Reply:
s/item/items/